CONTRIBUTING.md

```diff
@@ -44,6 +44,28 @@ for example:
 git clone git@github.com:YOUR-USERNAME/narwhals.git
 ```
 
+### 3. Add the `upstream` remote
+
+```bash
+git remote add upstream git@github.com:narwhals-dev/narwhals.git
```

Comment:
can we also add `git fetch upstream` underneath?

Reply:
done